src/librustdoc/json/conversions.rs

```diff
@@ -163,7 +163,9 @@ impl From<clean::ItemKind> for ItemEnum {
         use clean::ItemKind::*;
         match item {
             ModuleItem(m) => ItemEnum::ModuleItem(m.into()),
```

Comment:
ItemEnum should probably be renamed at some point, but no need to do it here.

Reply:
Maybe open an issue so we don't forget? (And mark it as `e-easy` too ;) )